test/validate/client.go

```diff
@@ -0,0 +1,37 @@
+package validate
+
+import (
+	"context"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+type Validator struct {
+	ctx         context.Context
+	clientset   *kubernetes.Clientset
+	config      *rest.Config
+	namespace   string
+	cni         string
+	restartCase bool
+}
+
+type IValidator interface {
+	ValidateStateFile() error
+	ValidateRestartNetwork() error
+	// ValidateDataPath() error
```

Comment:
It will be used by @jpayne3506 / @paulyufan for creating the data path test for different os. I added it as a comment so that we can add it later. Can add a todo to it if that makes more sense ?

Reply:
sure, it'll help track what it's for